docs/src/pages/hydroscope.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useEffect } from "react";
+import Layout from "@theme/Layout";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+import { Hydroscope, parseDataFromUrl } from "@hydro-project/hydroscope";
+
+function HydroscopePage() {
+  const [urlData, setUrlData] = useState<any>(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+
+  // Parse URL parameters on mount
+  useEffect(() => {
+    const parseUrlData = async () => {
+      try {
+        const searchParams = new URLSearchParams(window.location.search);
+        let dataParam = searchParams.get("data");
+        let compressedParam = searchParams.get("compressed");
+
+        // Fallback: also support hash fragment params (#data= / #compressed=)
+        if (!dataParam && !compressedParam && typeof window !== "undefined") {
+          const hash = window.location.hash?.replace(/^#/, "");
+          if (hash) {
+            const hashParams = new URLSearchParams(hash);
+            dataParam = hashParams.get("data") || dataParam;
+            compressedParam = hashParams.get("compressed") || compressedParam;
```

Comment:
Fallback to `dataParam`/`compressedParam` unnecessary, already known to be falsy/null
```suggestion
            dataParam = hashParams.get("data");
            compressedParam = hashParams.get("compressed");
```

Reply:
good catch!